test/e2e_node/image_list.go

```diff
@@ -52,7 +52,7 @@ const (
 // before test running so that the image pulling won't fail in actual test.
 var NodePrePullImageList = sets.NewString(
 	imageutils.GetE2EImage(imageutils.Agnhost),
-	"google/cadvisor:latest",
```

Comment:
curious why latest is not working now?

If this has an explicit version, please add a comment on how to pick the version to test with

Reply:
`google/cadvisor:latest` is a super old version. And it didn't have new changes for cgroupv2 support.  Therefore, picked up a latest version from https://github.com/google/cadvisor/releases. There is no explicit reason, recent cadvisor versions should all work

Reply:
Why is it the case? Can we make `:latest` to be actually latest

Reply:
Chatted with David,  he mentioned that it only pushes with specific version. so `:latest` doesn't work to be actually latest.  https://pantheon.corp.google.com/gcr/images/cadvisor/global/cadvisor

Reply:
can we fix the build to push `latest` tag instead? If it is not possible, can you please add a comment before this line explaining how this version was picked?

Reply:
If so, should we upgrade this image when cadvisor upgrades?

Then I would suggest to add the cadvisor version to https://github.com/kubernetes/kubernetes/blob/master//build/dependencies.yaml. So when we update go.mod for cadvisor and we have to update the image version here.

Reply:
Thanks for the comments, added the cadvisor version  to the https://github.com/kubernetes/kubernetes/blob/master//build/dependencies.yaml